### ISSUE TITLE
[Bug]: clear cache button on table inputs does not always or seldom works

### ISSUE THREAD

**Issue by Mattang-Dan** [apache/hop#3312]
### Apache Hop version?

2.7

### Java version?

microsoft-jdk-11.0.19-windows-x64.msi

### Operating system

Windows

### What happened?

The clear cache button on table inputs seldom works for me. I am connected to a remote SQL Server database and running Hop on a VM to Windows Server. I usually have to close Hop, and even VS Code (which launches Hop) for it to work.

launch script from VS Code.
```
#!/usr/bin/env pwsh
#Requires -Version 7

#Set Windows System Environment variables
$env:HOP_CONFIG = 'C:\dev\hop\HopBaseConfig\hop-config.json'
$env:HOP_CONFIG_FOLDER = 'C:\dev\hop\HopBaseConfig'

#Launch Hop 2.7
start-process -FilePath "C:\programs\hop\hop_2.7\hop-gui.bat" -WorkingDirectory "C:\programs\hop\hop_2.7"

```

### Issue Priority

Priority: 3

### Issue Component

Component: Hop Gui

**Comment by bamaer** [comment]
can you try setting this `SET use_cached_result = false` in the SQL statements in your advanced connection options to make sure this isn't related to JDBC driver caching (and not the Apache Hop cache)? 

https://learn.microsoft.com/en-us/azure/databricks/sql/admin/query-caching

**Comment by Mattang-Dan** [comment]
The only place I can think of adding it is here. Look right? this will affect every SQL server statement.

![image](https://github.com/apache/hop/assets/100213981/b74e867d-15ae-488f-b3c5-af2bc12715e4)

**Comment by bamaer** [comment]
nope, just in the Advanced tab, where it says "The SQL statements to run after connecting".
will be executed once every time a connection is created. that is not the same as every SQL server statement.

**Comment by Mattang-Dan** [comment]
I clicked the Test button in appropriate... but received an error:

![image](https://github.com/apache/hop/assets/100213981/1720f409-e884-4389-aafe-af425074fee5)

**Comment by hansva** [comment]
.take-issue
